Launcher/Views/Sidebar/CLSidebarView.swift

```diff
@@ -15,67 +15,64 @@ struct CLSidebarView: View {
 
     var body: some View {
         VStack(spacing: 0) {
-            ScrollView {
-                LazyVStack(spacing: 10) {
-                    ForEach(store.projects.sorted { a, b in
-                        a.created > b.created
-                    }, id: \.id) { p in
-                        HStack(spacing: 6) {
-                            CLProjectAvatarView(project: p, highlight: store.currentProjectID == p.id)
-                            VStack {
+            List {
```

Comment:
So the only change here is to use `List` than `ScrollView` and `LazyVStack`, and the other parts of diff are noises caused by indentation adjustments.

A question to make sure we understand the changes: does `List` already cover the lazy loading, or do you think lazy loading isn't needed?

Reply:
There is one sentence in  [wwdc20-10031](https://developer.apple.com/videos/play/wwdc2020/10031/?time=460): "List contents are always loaded lazily".